src/core.jl

```diff
@@ -73,12 +73,14 @@ function issubset{T<:SimpleGraph}(g::T, h::T)
 end
 
 
-"""Add `n` new vertices to the graph `g`."""
+"""Add `n` new vertices to the graph `g`. Returns true if all vertices
+were added successfully, false otherwise."""
 function add_vertices!(g::SimpleGraph, n::Integer)
+    added = true
     for i = 1:n
-        add_vertex!(g)
+        added &= add_vertex!(g)
```

Comment:
`add_vertex!` cannot really return false, therefore `add_vertices` could just return true always. Anyway the performance impact of this is irrelevant, so we can keep the check in case of any future change of the behaviour of `add_vertex!`